configs/katello/4.12.yaml

```diff
@@ -0,0 +1,116 @@
+---
+:project: katello
+:github_org: katello
+:nightly: false
+
+:releases:
+  :4.12.0:
+    :redmine_version_id: 1760
+:prior-releases:
+  :4.11.0:
+    :redmine_version_id: 1732
+  :4.10.1:
+    :redmine_version_id: 1774
+
+:code_name: Mile 42
+:mash_scripts:
```

Comment:
Like below, release engineering no longer uses this.

Reply:
Are you saying the entire :mash_scripts: section can be removed without complication?

Reply:
Removed